platypus/platypus.lua

```diff
@@ -143,29 +140,36 @@ function M.create(config)
 	local RAY_CAST_DOWN_ID = 6
 	local RAY_CAST_DOWN_LEFT_ID = 7
 	local RAY_CAST_DOWN_RIGHT_ID = 8
+ 
+	local RAYS = {}
+
+	-- Change the rays and bounds for collision detection
+	-- @param collisions table with top,left,right,bottom keys
+	function platypus.define_collisions(collisions)
```

Comment:
I suggest renaming this to set_collisions() as set and get are usually the standard naming convention for things such as this.

Reply:
ok, updated README.md too